src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -520,7 +520,7 @@ public void StopStake()
                 {
                     UnspentOutputs set = coinset.UnspentOutputs.FirstOrDefault(f => f?.TransactionId == infoTransaction.Transaction.Id);
                     TxOut utxo = (set != null) && (infoTransaction.Transaction.Index < set.Outputs.Length) ? set.Outputs[infoTransaction.Transaction.Index] : null;
-                    uint256 hashBock = this.chain.GetBlock((int) set.Height)?.HashBlock;
+                    uint256 hashBock = set != null ? this.chain.GetBlock((int) set.Height)?.HashBlock : null;
```

Comment:
Can you just rename this variable `hashBlock`? :stuck_out_tongue_winking_eye:

Reply:
Sure, fixed 🙂